components/global/core/io.cellery.observability.k8s.client/src/main/java/io/cellery/observability/k8s/client/ComponentPodsEventSource.java

```diff
@@ -63,14 +63,10 @@
 
     private static final Logger logger = Logger.getLogger(ComponentPodsEventSource.class.getName());
 
-    private static final String ATTRIBUTE_CELL = "cell";
-    private static final String ATTRIBUTE_COMPONENT = "component";
     private static final String ATTRIBUTE_POD_NAME = "name";
```

Comment:
If you are moving the attribute name constants to a separate Constants class, better to move all of them to be consistent.